setup.py

```diff
@@ -570,26 +570,46 @@ def determine_gl_flags():
 
 def determine_sdl2():
     flags = {}
+    # This is redundant; the only time determine_sdl2 is called is preceded by
+    # verification that this condition is already satisfied
     if not c_options['use_sdl2']:
         return flags
 
+    # Check the environment for a Kivy SDL2 path (all platforms) or for Include
+    # and Lib (intended to catch Windows installation, see #4941)
```

Comment:
I disagree what constitutes "obvious" in the context of a highly convoluted setup, but I will respect the maintainers' wishes and remove comments (when I get a chance to do so, as this already chewed up half of my afternoon, because the code was utterly unintelligible to a professional dev but casual contributor)